src/rosdiscover/cli.py

```diff
@@ -26,7 +26,7 @@ def _launch(config: Config) -> SystemSummary:
                                ) as interpreter:
```

Comment:
We could improve this by replacing `Interpreter.for_image` with `Interpreter.for_config`. (Alternatively, we could be more OO-friendly by adding an `interpreter()` method to `Config`, that produces an interpreter for a given configuration.)